tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -696,11 +696,187 @@ void EXPECT_SIZEEQ(CGSize pathSize, CGFloat width, CGFloat height) {
     EXPECT_POINTEQ(boundingBox.origin, 75, 75);
     EXPECT_SIZEEQ(boundingBox.size, 25, 25);
 
-    CGPathAddPath(path, nullptr, secondPath);
+    CGPathAddPath(path, NULL, secondPath);
     boundingBox = CGPathGetBoundingBox(path);
```

Comment:
Consistency with the rest of the file. I'll change them to nullptr instead then.

---
In reply to: [85629785](https://github.com/Microsoft/WinObjC/pull/1255#discussion_r85629785) [](ancestors = 85629785)